fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java

```diff
@@ -951,6 +951,10 @@ public Lao getCurrentLaoValue() {
     return mCurrentLao.getValue();
   }
 
+  public void setCurrentLao(Lao lao) {
```

Comment:
I guess this is a solution. It could be a good idea to add 
```suggestion
  @VisibleForTesting
  public void setCurrentLao(Lao lao) {
```

So that the method is only visible for testing.